daktari/checks/android.py

```diff
@@ -0,0 +1,25 @@
+from daktari.check import Check, CheckResult
+from daktari.os import OS, get_env_var_value
+
+
+class AndroidNdkHomeSet(Check):
+    def __init__(self, expected_version):
+        self.name = "env.variableSet.androidNdkHome"
+        self.variable_name = "ANDROID_NDK_HOME"
+        self.expected_version = expected_version
+        self.suggestions = {
+            OS.GENERIC: f"""
+            Export {self.variable_name} in your shell config.
+            The expected value is ANDROID_SDK_HOME/ndk/{self.expected_version}.
+            If you manage your android sdk using Android Studio, you can find your ANDROID_SDK_HOME by going to
```

Comment:
I'm hoping anyone who has decided to manage their android sdk manually will know where their ANDROID_SDK_HOME path is. Not sure there's much we can do to help them if not!